src/lib.rs

```diff
@@ -33,8 +33,9 @@ extern crate hyper;
 extern crate "typemap" as tmap;
 extern crate plugin;
 extern crate "modifier" as modfier;
-extern crate error;
+extern crate "error" as err;
```

Comment:
Hmm.. I think it's usually not needed - I only did it here so I could have a module named error as well.